src/Files.App/UserControls/AddressToolbar.xaml

```diff
@@ -425,7 +424,6 @@
 				MinHeight="33"
 				HorizontalAlignment="Stretch"
 				VerticalAlignment="Center"
-				extensions:FrameworkElementExtensions.Cursor="Hand"
```

Comment:
Is there a replacement?

Reply:
No, unfortunately.

Reply:
Can you create an issue tracking the changes that will need to be followed up on for this PR?